ods_ci/tests/Resources/Files/triton/triton_tensorflow_modelmesh_runtime.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: serving.kserve.io/v1alpha1
```

Comment:
this is the same as https://github.com/red-hat-data-services/ods-ci/pull/1835/files
Can you avoid duplication and use one single runtime definition for multiple models?

Reply:
@Raghul-M you marked this as resolved, how this was resolved?